build failture with wasmtime

```
vscode@docker-desktop:~/proxy-wasm-cpp-host$ bazel build --define=engine=multi :lib
INFO: Analyzed target //:lib (0 packages loaded, 0 targets configured).
INFO: Found 1 target...
ERROR: /build/proxy-wasm-cpp-host/BUILD:246:11: Compiling src/wasmtime/prefixed_wasmtime.cc failed: (Exit 1): gcc failed: error executing command (from target //:prefixed_wasmtime_lib) /usr/bin/gcc -U_FORTIFY_SOURCE -fstack-protector -Wall -Wunused-but-set-parameter -Wno-free-nonheap-object -fno-omit-frame-pointer '-std=c++14' -MD -MF ... (remaining 29 arguments skipped)

Use --sandbox_debug to see verbose messages from the sandbox and retain the sandbox build root for debugging
In file included from bazel-out/aarch64-fastbuild/bin/src/wasmtime/prefixed_wasmtime.cc:27:
bazel-out/aarch64-fastbuild/bin/src/wasmtime/prefixed_types.h:16:10: fatal error: crates/c-api/include/prefixed_wasm.h: No such file or directory
   16 | #include "crates/c-api/include/prefixed_wasm.h"
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
Target //:lib failed to build
Use --verbose_failures to see the command lines of failed build steps.
INFO: Elapsed time: 8.030s, Critical Path: 7.29s
INFO: 454 processes: 10 internal, 1 local, 443 processwrapper-sandbox.
FAILED: Build did NOT complete successfully
```

--- a/bazel/external/wasmtime.BUILD
+++ b/bazel/external/wasmtime.BUILD
@@ -21,7 +21,7 @@ genrule(
         "crates/c-api/include/wasm.h",
     ],
     outs = [
-        "include/prefixed_wasm.h",
+        "crates/c-api/include/prefixed_wasm.h",
     ],
     cmd = """
         sed -e 's/\\ wasm_/\\ wasmtime_wasm_/g' \

